models/board/src/index.ts

```diff
@@ -414,6 +414,23 @@ export function createModel (builder: Builder): void {
   builder.mixin(board.class.Card, core.class.Class, view.mixin.IgnoreActions, {
     actions: [view.action.Delete, task.action.Move]
   })
+
+  createAction(
+    builder,
+    {
+      action: board.actionImpl.ConvertToCard,
+      label: board.string.ConvertToCard,
+      icon: board.icon.Card,
+      category: board.category.Card,
+      query: {
+        attachedToClass: task.class.TodoItem
```

Comment:
Can we add a TODO comment to properly update the query in the future?

Reply:
Added